src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ReaderServiceImplTest {
+    private static final ReaderService READER_SERVICE = new ReaderServiceImpl();
```

Comment:
```suggestion
    private static final ReaderService readerService = new ReaderServiceImpl();
```
this class is not deeply immutable that's why we use regular naming here